src/StockServiceManager.php

```diff
@@ -2,201 +2,62 @@
 
 namespace Drupal\commerce_stock;
 
+use Drupal\commerce\Context;
 use Drupal\commerce\PurchasableEntityInterface;
-use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\commerce_stock\Resolver\ChainStockServiceResolverInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
 
 /**
- * The stock service manager, responsible for handling services and transactions.
- *
- * @see StockAvailabilityChecker.
- *
- * @package Drupal\commerce_stock
+ * Commerce stock service manager.
  */
-class StockServiceManager implements StockServiceManagerInterface, StockTransactionsInterface {
```

Comment:
Why is the StockTransactionsInterface removed? it is currently used e.g.  **$this->stockServiceManager->createTransaction()**